app/Jobs/ImportFacebookSharePosts.php

```diff
@@ -78,9 +78,8 @@ public function handle(Rogue $rogue)
                 'status' => 'accepted',
                 'source' => 'importer-client',
                 'source_details' => json_encode(['original-source' => $record['event.source']]),
-                'created_at' => strtotime($record['to_timestamp']),
+                'created_at' => date(DATE_ATOM, strtotime($record['to_timestamp'])),
```

Comment:
@chloealee what's the `to_timestamp` column look like?

Reply:
@katiecrane here's an example: `6/8/2018 18:27:00`

Reply:
ohhh @katiecrane maybe I should do something like you and @sbsmith86 did in your imports:
```
strtolower(Carbon::parse($record['Started registration'])->format('F-Y'));
```

Reply:
oh wait, I think I am wrong I don't think we want to format that like that here.

Reply:
I think you might be able to do something like this instead if you want to use Carbon:
```
Carbon::parse($record['to_timestamp'])->toDateTimeString();
```
Does that give you the right format?

Reply:
ah yes it does! Thanks @katiecrane ! Updated - I like that way better :)